### PR TITLE
Update to latest versions and add CI

### PR DESCRIPTION
**Pull request by ekrich**

### CODE REVIEW

**Comment by ekrich** [comment]
Related for updates: https://github.com/VirtusLab/scala-steward-repos/pull/167

File: project/giter8.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.14.0")
+addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.16.0")
```
Comment (LeeTibbert) [inline project/giter8.sbt:1]: I believe this sbt-giter8 should be, as of two months ago, 0.16.1.  If there was a reason for the G8 folks to go to .1,
probably worth using it.

File: src/main/g8/build.sbt [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
 enablePlugins(ScalaNativePlugin)
+
+// set to Debug for compilation details (Info is default)
+logLevel := Level.Info
+
+// import to add Scala Native options
+import scala.scalanative.build._
+
+// defaults set with common options shown
+nativeConfig ~= { c =>
```
Comment (LeeTibbert) [inline src/main/g8/build.sbt:12]: I like the way you handled the `nativeConfig ~=` section and will probably use the snippet in my own local.sbt.

File: project/giter8.sbt (hunk 1 above)
Comment (ekrich) [reply project/giter8.sbt]: Thanks.

File: src/main/g8/build.sbt (hunk 2 above)
Comment (ekrich) [reply src/main/g8/build.sbt:12]: I want to make the next steps or experimentation easier. I appreciate the 👍

**Review by WojciechMazur — APPROVED** [review]

**Pull request merged**